pom.xml

```diff
@@ -1348,6 +1348,40 @@ flexible messaging model and an intuitive client API.</description>
             </execution>
         </executions>
       </plugin>
+      <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
```

Comment:
if we can change the release procedure I would like to add this to the 'apache-release' profile

Reply:
actually we need this source distribution in order to build the rpm and deb packages during tests, so it is better to leave this operation active for every build, as we are currently doing